packages/rocketchat-apps/client/admin/appManage.html

```diff
@@ -10,6 +10,9 @@
 					<span class="rc-switch__button">
 						<span class="rc-switch__button-inside"></span>
 					</span>
+					<span class="rc-switch__text">
+						{{_ "Activate"}}
```

Comment:
I think we should toggle this text

Reply:
Since layout only stated 'Enable' and I didn't want to add yet another i18n string, I thought this would be enough.